src/System.Linq/tests/EnumerableTests.cs

```diff
@@ -0,0 +1,131 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using Xunit;
+
+namespace System.Linq.Tests
+{
+    public partial class EnumerableTests
+    {
+        [Fact]
+        public void Average()
+        {
+            var array = Enumerable.Range(1, 10).ToArray();
+            var min = array.Min();
+            Assert.Equal(min, 1);
+        }
+
+        [Fact]
+        public void All()
+        {
+            var array = Enumerable.Range(1, 10).ToArray();
+            Assert.True(array.All(i => i > 0));
+            Assert.False(array.All(i => i > 1));
```

Comment:
I don't think you actually need to do this 10 times; it should be sufficient to test a true and a false case.  If you really wanted to do it for all of the values, though, it'd be cleaner to have a for loop around a single Assert.False rather than duplicating it for each value.